src/Jobs/Product/UpdateProductJob.php

```diff
@@ -50,9 +50,8 @@ public function failed(Throwable $throwable): void
             ->on($this->product)
             ->useLog('error')
             ->withProperties([
-                'message' => $throwable->getMessage(),
                 'code' => $throwable->getCode(),
```

Comment:
I don't think the code of the exception is that valueable, we could omit the properties

Reply:
It is the HTTP status code that Akeneo returns which could be helpful